bin/fontbakery-fix-nameids.py

```diff
@@ -59,6 +62,14 @@ def drop_superfluous_mac_names(ttfont):
                 ttfont['name'].names.remove(name)
 
 
+def drop_mac_names(ttfont):
+    """Drop all mac names"""
+    for n in range(255):
+        name = ttfont['name'].getName(n, 1, 0, 0)
+        if name:
+            ttfont['name'].names.remove(name)
```

Comment:
note to self: the ttLib name table class has `getName` and `setName` methods; I guess for consistency we could have a `delName` too.